search-app/src/cmr/search/web/routes.clj

```diff
@@ -0,0 +1,68 @@
+(ns cmr.search.web.routes
+  "This namespace is the one responsible for combining the routes intended for
+  use by libraries and applications (API routes) and the routes intended for
+  human consumption (site routes). It also provides routes that apply to both
+  (e.g., robots.txt)."
+  (:require
+   ;; Third-party libs
+   [clojure.java.io :as io]
+   [compojure.core :refer :all]
+   [compojure.route :as route]
+   [ring.middleware.keyword-params :as keyword-params]
+   [ring.middleware.nested-params :as nested-params]
+   [ring.middleware.params :as params]
+
+   ;; CMR libs
+   [cmr.acl.core :as acl]
+   [cmr.common.api.errors :as errors]
+   [cmr.common.api.context :as context]
+   [cmr.common-app.api.routes :as common-routes]
+   [cmr.search.api.request-context-user-augmenter :as context-augmenter]
+   [cmr.search.api.routes :as api-routes]
+   [cmr.search.site.routes :as site-routes]))
```

Comment:
I kind of like the way you have separated the requires into CMR/non-CMR, but the standard we use is to sort alphabetically (the cmr ones end up together that way anyhow). I'm not opposed to switching to this, but we should be consistent.